tox.ini

```diff
@@ -217,3 +217,4 @@ junit_suite_name = nimi-python
 python_files = *.py
 
 
+
```

Comment:
What's this for?

Reply:
See https://docs.pytest.org/en/latest/reference.html#confval-python_files

Since we put some unit tests into file not named `test_*.py` or `*_test.py` we need to override what files pytest looks in.